diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/classes/CompactInitialization.kt

```diff
@@ -106,6 +108,14 @@ class CompactInitialization(private val configRules: List<RulesConfig>) : Rule("
                             bodyExpression.addChild(it.clone() as ASTNode, null)
```

Comment:
I think it would be more consistent, if we change this `forEach` to `forEachIndexed` and inside it `if (idx == 0 && elementType == WHITE_SPACE)` add node to parent, otherwise add to `bodyExpression`.